### DISCUSSION TITLE
' TORCH_USE_CUDA_DSA" Help please

### DISCUSSION THREAD

**Discussion by Gofer997** [w-okada/voice-changer#793] [Q&A]
![IMG_20230905_171043_346](https://github.com/w-okada/voice-changer/assets/144127888/77ec9504-9408-4635-b8ed-9d388798c386)
stuck (last line) and no sound

**Comment by Evexal** [comment]
As i remember you need install pytorch, you can type in google pytorch and it's prob gonna be first site. And just follow instructions, or maybe you use wrong version.
